docs/claude-agent-sdk-docs/cost-tracking.md

```
@@ -223,7 +223,38 @@ The Agent SDK automatically uses [prompt caching](https://platform.claude.com/do
+  )
+  ```
+
+  ```typescript TypeScript theme={null}
+  const options = {
+    env: {
+      ...process.env,
+      CLAUDE_CODE_USE_BEDROCK: "1",
+      ENABLE_PROMPT_CACHING_1H: "1",
+    },
+  };
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)